src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -210,6 +210,18 @@
   <data name="SARIF1019_RuleIdMustBePresentAndConsistent" xml:space="preserve">
     <value>In every result, at least one of the properties result.ruleId and result.rule.id must be present. If both are present, they must be equal.</value>
   </data>
+  <data name="SARIF1022_PreferMessageIdInsteadOfMessageText" xml:space="preserve">
+    <value>Prefer message.id instead of message.text</value>
+  </data>
+  <data name="SARIF1022_ResultShouldUseMessageIdInsteadOfMessageText" xml:space="preserve">
+    <value>{0}: The result should use message.id instead of message.text value '{1}'</value>
+  </data>
+  <data name="SARIF1022_RuleMetadataDoesntIncludeMessageId" xml:space="preserve">
+    <value>{0}: The rule metadata doesn't include the message id value '{1}'</value>
+  </data>
+  <data name="SARIF1022_RuleMetadataIsNotConsistentWithArguments" xml:space="preserve">
```

Comment:
>RuleMetadataIsNotConsistentWithArguments [](start = 24, length = 40)

`SARIF1011_IncorrectResultMessageArgumentsCount`.

Reply:
(still active)

---
In reply to: [434713964](https://github.com/microsoft/sarif-sdk/pull/1901#discussion_r434713964) [](ancestors = 434713964)